docs/notebooks/Protein_Structure_Alignment.ipynb

```diff
@@ -457,22 +457,22 @@
-      "  PDBConstructionWarning,\n",
-      "/home/legend/softs/miniconda3/envs/PA_py37/lib/python3.7/site-packages/Bio/PDB/StructureBuilder.py:92: PDBConstructionWarning: WARNING: Chain D is discontinuous at line 5167.\n",
-      "  PDBConstructionWarning,\n",
-      "/home/legend/softs/miniconda3/envs/PA_py37/lib/python3.7/site-packages/Bio/PDB/StructureBuilder.py:92: PDBConstructionWarning: WARNING: Chain A is discontinuous at line 5211.\n",
-      "  PDBConstructionWarning,\n",
-      "/home/legend/softs/miniconda3/envs/PA_py37/lib/python3.7/site-packages/Bio/PDB/StructureBuilder.py:92: PDBConstructionWarning: WARNING: Chain B is discontinuous at line 5269.\n",
-      "  PDBConstructionWarning,\n",
-      "/home/legend/softs/miniconda3/envs/PA_py37/lib/python3.7/site-packages/Bio/PDB/StructureBuilder.py:92: PDBConstructionWarning: WARNING: Chain C is discontinuous at line 5327.\n",
-      "  PDBConstructionWarning,\n",
-      "/home/legend/softs/miniconda3/envs/PA_py37/lib/python3.7/site-packages/Bio/PDB/StructureBuilder.py:92: PDBConstructionWarning: WARNING: Chain D is discontinuous at line 5385.\n",
-      "  PDBConstructionWarning,\n"
+      "/home/legend/softs/miniconda3/envs/PA_py38/lib/python3.8/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain A is discontinuous at line 5037.\n",
+      "  warnings.warn(\n",
+      "/home/legend/softs/miniconda3/envs/PA_py38/lib/python3.8/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain B is discontinuous at line 5080.\n",
+      "  warnings.warn(\n",
+      "/home/legend/softs/miniconda3/envs/PA_py38/lib/python3.8/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain C is discontinuous at line 5124.\n",
+      "  warnings.warn(\n",
+      "/home/legend/softs/miniconda3/envs/PA_py38/lib/python3.8/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain D is discontinuous at line 5167.\n",
+      "  warnings.warn(\n",
+      "/home/legend/softs/miniconda3/envs/PA_py38/lib/python3.8/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain A is discontinuous at line 5211.\n",
+      "  warnings.warn(\n",
+      "/home/legend/softs/miniconda3/envs/PA_py38/lib/python3.8/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain B is discontinuous at line 5269.\n",
+      "  warnings.warn(\n",
+      "/home/legend/softs/miniconda3/envs/PA_py38/lib/python3.8/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain C is discontinuous at line 5327.\n",
+      "  warnings.warn(\n",
+      "/home/legend/softs/miniconda3/envs/PA_py38/lib/python3.8/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain D is discontinuous at line 5385.\n",
+      "  warnings.warn(\n"
```

Comment:
These changes (and similar ones below) should not be committed or staged. Please remove them from the commit and force-push to your `fwmeng88:protein_align` branch.